modules/common/endpoint/endpoint.go

```diff
@@ -184,7 +203,6 @@ func ExposeEndpoints(
 						ServiceName:    endpointName,
 						TargetPortName: endpointName,
 					}),
-					exportLabels,
 					timeout,
 					data.RouteOverride,
```

Comment:
@stuggi quick question here: when a `GenericRoute` is created, in case `tls` is used we're terminating the ssl on the Route, w/o re-encrypting the second segment (from the `Route` to the `Service`). 
Is my understanding correct? 
This would be a good behavior to keep and provide parameters via `RouteOverride` to change the behavior to `re-encrypt`. 
For instance Glance atm doesn't support tls, and in TripleO we used to deploy an httpd in front of GlanceAPI to properly handle tls.
This is something I'm going to work on in the next few weeks, but wanted to check that we actually have the ability to terminate the connection on the `Route` until that part is done.

Reply:
yes, I'll wait to see the first patch series landed, and when doing tls we can sync up and discuss about the Glance use case. We might need to have an additional sidecar container w/ httpd or make it default like it is today in Manila/Cinder.